test/minter/mintVRGDA.t.sol

```diff
@@ -161,16 +164,13 @@ contract Mint is BaseTest {
         cheats.stopPrank();
 
         // TEST: check that fixed price is set to 0 after setting VRGDA
-        minter.setVRGDAForMintConfig(mintConfigId, targetPrice, priceDecayPercent, perTimeUnit);
+        minter.setVRGDAForMintConfig(mintConfigId, targetPrice, priceDecayPercent, perTimeUnit, secondsPerTimeUnit);
```

Comment:
why is this part of this test? should be in the setVRGDAForMintConfig test

Reply:
I need to set the config for testing the vrgda

Reply:
Also I see we don't have test for the setVRGDA config. It would be nice to move this one there and add some more 👌